### DISCUSSION TITLE
What's the equivalent of Guidance Scale and Sharpness in automatic 1111?

### DISCUSSION THREAD

**Discussion by ekimia** [lllyasviel/Fooocus#1738] [Q&A] [answered]
Not having much luck finding equivalent settings - it does seem ADM Guidance == CFG Scale in a1111

**Comment by lllyasviel** [comment, marked as answer]
Fooocus is designed for best result quality rather than following A1111 parameter standards.

Fooocus Guidance Scale ≈ A1111 CFG + cfg rescale (similar to https://github.com/mcmonkeyprojects/sd-dynamic-thresholding)

Fooocus Sharpness ≈ A1111 additinal noise (https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/12564) + SAG (https://github.com/ashen-sensored/sd_webui_SAG) + some other anisotropic tricks from me

Users who do not completely understand these should view A1111 and Fooocus as different software and use them independently.
